### DISCUSSION TITLE
Is there something wrong with my project project id? Why can't I connect now?

### DISCUSSION THREAD

**Discussion by lrlazy** [WalletConnect/walletconnect-monorepo#2942] [Developer Support]
Is there something wrong with my project project id? Why can't I connect now?
I was able to connect and transfer money when I tested it before, but now I can’t connect no matter what
parijectId : 419f8f621597f7fc5e574bc20df24180

**Comment by chadyj** [comment]
Hey we'd be happy to help, but can you please let us know what wallet and/or dapp you are trying to connect with?

There are no restrictions on your project, so it must be a wallet/dapp issue.

**Reply by lrlazy** [reply]
alphaWallet

**Reply by lrlazy** [reply]
I've connected successfully before, but I can't connect anymore.
